spec/file_system_data_spec.rb

```diff
@@ -67,15 +67,16 @@
     after { TestCodebase.teardown }
 
     it '#get' do
-      data.config = { read: ['a.yml', '{b,c}.yml'] }
+      data.config = { read: ['a.yml', '{b,c}.yml', 'd.yml'] }
       TestCodebase.setup(
         'a.yml' => { en: { a: 1 } }.stringify_keys.to_yaml,
         'b.yml' => { en: { b: 1 } }.stringify_keys.to_yaml,
-        'c.yml' => { en: { c: 1 } }.stringify_keys.to_yaml
+        'c.yml' => { en: { c: 1 } }.stringify_keys.to_yaml,
+        'd.yml' => "---\n:en:\n :d: &d test\n :d_alias: *d\n"
```

Comment:
This could be made a bit more readable by using a heredoc:

```suggestion
        'd.yml' => <<<~YAML
          ---
          :en:
            :d: &d test
            :d_alias: *d
        YAML
```

Reply:
Noted - I was trying to match the single-line format of the hashes!
Clarified and simplified the HEREDOC YAML to match real-life YAML structures